Frontend/ui-driver/src/Helpers/Utils.purs

```diff
@@ -429,9 +466,9 @@ getVehicleVariantImage variant =
                         "SUV"  -> "ny_ic_suv_ac_side," <> commonUrl <> "ny_ic_suv_ac_side.png"
                         _      -> "ny_ic_sedan_ac_side," <> commonUrl <> "ny_ic_sedan_ac_side.png"
         _          -> case variant of
-                        "SEDAN"     -> "ic_sedan," <> url <> "ic_sedan.png"
-                        "SUV"       -> "ic_suv," <> url <> "ic_suv.png"
-                        "HATCHBACK" -> "ic_hatchback," <> url <> "ic_hatchback.png"
+                        "SEDAN"     -> "ny_ic_sedan_car_side," <> url <> "ny_ic_sedan_car_side.png"
+                        "SUV"       -> "ny_ic_suv_car_side," <> url <> "ny_ic_suv_car_side.png"
+                        "HATCHBACK" -> "ny_ic_hatchback_car_side," <> url <> "ny_ic_hatchback_car_side.png"
```

Comment:
@ashkriti is the PR for asset store merged?

Reply:
yes